packages/suite/src/views/wallet/tokens/TokensNavigation.tsx

```diff
@@ -1,104 +1,31 @@
 import { Dispatch, SetStateAction, useEffect, useState } from 'react';
-import styled, { css } from 'styled-components';
 
-import { TranslationKey } from '@suite-common/intl-types';
-import { Route } from '@suite-common/suite-types';
 import { SelectedAccountLoaded } from '@suite-common/wallet-types';
 import { selectCoinDefinitions } from '@suite-common/token-definitions';
-import { Elevation, borders, spacingsPx, mapElevationToBorder, typography } from '@trezor/theme';
-import { IconButton, IconName, useElevation, variables } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+import { IconButton } from '@trezor/components';
 import { EventType, analytics } from '@trezor/suite-analytics';
 
 import { useDispatch, useSelector } from 'src/hooks/suite';
-import { NavigationItem } from '../../../../../components/suite/layouts/SuiteLayout/Sidebar/NavigationItem';
+import { NavigationItem } from 'src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem';
 import { getTokens } from 'src/utils/wallet/tokenUtils';
 import { selectIsDebugModeActive } from 'src/reducers/suite/suiteReducer';
 import { SearchAction } from 'src/components/wallet/SearchAction';
 import { openModal } from 'src/actions/suite/modalActions';
+import { Divider } from '@trezor/components';
```

Comment:
Add these imports to the import above

Reply:
I combined the imports where it was possible, did you mean something else?